src/interactive/scala/tools/nsc/interactive/CompilerControl.scala

```diff
@@ -288,6 +290,8 @@ trait CompilerControl { self: Global =>
     accessible: Boolean,
     inherited: Boolean,
     viaView: Symbol) extends Member {
+    // should be a case class parameter, but added as a var instead to preserve compatibility with the IDE
+    var prefix: Type = NoType
```

Comment:
Yeah, I think you do. We'd need to phase that out first
On Thu, 24 Sep 2015 at 6:05 PM, Iulian Dragos <notifications@github.com>
wrote:

> In src/interactive/scala/tools/nsc/interactive/CompilerControl.scala
> <https://github.com/scala/scala/pull/4725#discussion_r40294033>:
>
> > @@ -288,6 +290,8 @@ trait CompilerControl { self: Global =>
> >      accessible: Boolean,
> >      inherited: Boolean,
> >      viaView: Symbol) extends Member {
> > +    // should be a case class parameter, but added as a var instead to preserve compatibility with the IDE
> > +    var prefix: Type = NoType
>
> I think source-compatibility should be ok... Do we pattern match on this?
> :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scala/scala/pull/4725/files#r40294033>.
>